doc/communication_team_emeritus.rst

```diff
@@ -0,0 +1,12 @@
+.. raw :: html
+
+    <!-- Generated by generate_authors_table.py -->
+    <div class="sk-authors-container">
+    <style>
+      img.avatar {border-radius: 10px;}
+    </style>
+    <div>
+    <a href='https://github.com/reshamas'><img src='https://avatars.githubusercontent.com/u/2507232?v=4' class='avatar' /></a> <br />
+        <p>Reshama Shaikh</p>
+    </div>
+    </div>
```

Comment:
authors emeritus is a simple list. Can we do the same here ? the artifacts looks a bit weird

Reply:
In this case:

```suggestion
- Reshama Shaikh
```